tests/integration/API/JSON/v1/FormsTest.php

```diff
@@ -19,16 +21,20 @@ class FormsTest extends \MailPoetTest {
   /** @var Forms */
   private $endpoint;
 
+  /** @var FormsRepository */
+  private $formsRepository;
+
   /** @var WPFunctions */
   private $wp;
 
   public function _before() {
     parent::_before();
     $this->endpoint = ContainerWrapper::getInstance()->get(Forms::class);
+    $this->formsRepository = ContainerWrapper::getInstance()->get(FormsRepository::class);
     $this->wp = WPFunctions::get();
-    $this->form1 = Form::createOrUpdate(['name' => 'Form 1']);
-    $this->form2 = Form::createOrUpdate(['name' => 'Form 2']);
-    $this->form3 = Form::createOrUpdate(['name' => 'Form 3']);
+    $this->form1 = $this->createForm('Form 1');
+    $this->form2 = $this->createForm('Form 2');
+    $this->form3 = $this->createForm('Form 3');
     Segment::createOrUpdate(['name' => 'Segment 1']);
```

Comment:
Could you please do the same small factory method for Segment the same way you did for the form here? It would be nice to clean the test from the usage of old models.

Reply:
Sure, done